timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -269,6 +276,7 @@ export class TimeGraphChart extends TimeGraphChartLayer {
     }
 
     protected addRow(row: TimelineChart.TimeGraphRowModel, height: number, rowIndex: number) {
+        console.log("Po ra Daffa Nuvvu cheppedhi nenenti vinedhi");
```

Comment:
2 possible translations: "Po ra daffa I hear what you say" or "Poe's what i have to say you're wrong, vinedi". Sounds right?

Reply:
Sorry for the mess. Cleared it. :/

Reply:
Ha. Satish dropped some Telugu on us. @muddana-satish do at least tell us what it means. The suspense is killing me

Reply:
It means: _Go Away! Why should I listen to you._ 
That's me interacting with JS engine.